RELEASE-NOTES.txt

```diff
@@ -4,6 +4,8 @@
 
 ### Enhancements
 
+ - Updated menu icon to use the new icon set [#1694](https://github.com/Automattic/simplenote-electron/pull/1694)
```

Comment:
Is this so much an _enhancement_ as it is _other changes_? 😉 joking-not-joking

Reply:
I think it is an enhancement since we are enhancing the UI by updating the icons.